candidate/models.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from ballot_measure.models import BallotMeasureChoice
+
+
+class SocialMediaModel(models.Model):
+    class Meta:
+        abstract = True
+
+    photo_url = models.ImageField(blank=True)
+    website_url = models.URLField(
+        help_text='URL for the official website.', blank=True)
+    facebook_url = models.URLField(
+        help_text='URL for the Facebook page.', blank=True)
+    twitter_url = models.URLField(
+        help_text='URL for the Twitter page.', blank=True)
+
+
+@python_2_unicode_compatible
+class Party(SocialMediaModel):
+    """
+    """
+    name = models.CharField(
+        max_length=255, help_text='The party name.')
+
+    def __str__(self):
+        return "The %s party" % self.name
```

Comment:
A lot of local candidates aren't party-affiliated, as I understand. Glad to see you made the Person's `party` foreign key nullable.